src/os/ui/featureedit.js

```diff
@@ -980,8 +1010,20 @@ os.ui.FeatureEditCtrl.prototype.loadFromFeature_ = function(feature) {
       if (coordinate) {
         this['pointGeometry'] = {
           'lon': coordinate[0],
-          'lat': coordinate[1]
+          'lat': coordinate[1],
+          'alt': coordinate[2]
```

Comment:
Use `coordinate[2] || 0` to ensure you get a numeric value. If a coordinate does not specify an altitude, you can assume a value of 0.